src/main/java/eu/europeana/metis/sandbox/config/amqp/CreatedQueueConfig.java

```diff
@@ -43,9 +43,9 @@ public CreatedQueueConfiguration(MessageConverter messageConverter) {
   SimpleRabbitListenerContainerFactory externalValidationFactory(
       SimpleRabbitListenerContainerFactoryConfigurer configurer,
       ConnectionFactory connectionFactory) {
-    super.setConcurrentQueueConsumers(concurrentConsumers);
```

Comment:
I see here the mistake. it will change to the parent class, but not for the actual class right?